doc/website/blog/2022-03-14-ent-hooks-examples.md

```diff
@@ -0,0 +1,430 @@
+---
+title: A few examples for Ent hooks  
```

Comment:
would you click a post with this title? lets find something that isn't cringey click-baitey but still is attractive

Reply:
👍